lib/src/main/java/org/team100/lib/motor/drive/NeoDriveMotor.java

```diff
@@ -77,7 +79,8 @@ public NeoDriveMotor(
             int currentLimit,
             double gearRatio,
             double wheelDiameter,
-            double kV) {
+            double kV,
+            PIDController lowLevelVelocityController) {
```

Comment:
don't pass a *controller* here since you're not using it to control anything, and you're also not listening for updates to the parameters through smart dashboard.  just pass the coefficients, or make a config object for them like your SysParam idea.